compiler/dalgona/CMakeLists.txt

```diff
@@ -3,14 +3,19 @@
 #   Ubuntu20.04; default python3.8
 #   Ubuntu22.04; default python3.10
 #   refer https://github.com/Samsung/ONE/issues/9962
-find_package(PythonInterp 3.8 QUIET)
-find_package(PythonLibs 3.8 QUIET)
+find_package(Python3 3.8 QUIET COMPONENTS Interpreter Development)
 
-if(NOT ${PYTHONINTERP_FOUND})
+if(NOT Python3_FOUND)
```

Comment:
`Python3_FOUND` requires cmake 3.12 or higher. (https://cmake.org/cmake/help/latest/module/FindPython3.html)

I'm not sure if we can use `cmake_minimum_required(VERSION 3.12)` for this tool.
@seanshpark @hseok-oh 
Is it OK to use `Python3_FOUND` instead of `PYTHONINTERP_FOUND` on other ubuntu versions?

Reply:
@jinevening , PTAL